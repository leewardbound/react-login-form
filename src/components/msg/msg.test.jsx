import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Msg from '.';

const renderer = new ShallowRenderer();

describe('Msg', function () {
  it('should render correctly', function () {
    ['success', 'error'].forEach(function (type) {
      const component = renderer.render(
        <Msg
          type={type}
        >
          <h1>Lorem ipsum</h1>
        </Msg>
      );

      expect(component).toMatchSnapshot();
    });
  });
});
