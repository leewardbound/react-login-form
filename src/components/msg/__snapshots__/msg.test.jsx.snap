// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Msg should render correctly 1`] = `
<styled.p
  type="success"
>
  <h1>
    Lorem ipsum
  </h1>
</styled.p>
`;

exports[`Msg should render correctly 2`] = `
<styled.p
  type="error"
>
  <h1>
    Lorem ipsum
  </h1>
</styled.p>
`;
